test/run

```diff
@@ -143,24 +161,32 @@ def print_counter(counter):
 
 
 def main():
+    args = get_args()
     logging.basicConfig(format="run-tests: %(message)s")
     test_dir = pathlib.Path(__file__).resolve().parent
     root = test_dir.parent
     litani = root / "litani"
+    if args.output_dir:
+        output_dir = pathlib.Path(args.output_dir) / str(uuid.uuid4())
+        output_dir.mkdir(exist_ok=True, parents=True)
+        os.chdir(output_dir)
+    else:
+        output_dir = test_dir / "output"
```

Comment:
It appears as though my previous comment was not addressed. Could you please explain? I believe that `args.output_dir` should have a default value of `test_dir / "output"` and when a user specifies a different output directory, that should be set as the output directory. Why do we create a need to make a subdirectory with a UUID for that directory?

Reply:
Resolved